CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.5.1](https://github.com/ASFHyP3/hyp3-sdk/compare/v0.5.0...v0.5.1)
+
+## Changed
+- `util.AUTH_URL` changed so that no login page is loaded, and a 401 is given on auth failure.
```

Comment:
Maybe this?  At a minimum, `Changed` should be H3
### Fixed
`get_authenticated_session` now correctly throws `AuthenticationError` when no `.netrc` file exists and no credentials are provided